extras/pf-cmake/auto.cmake

```diff
@@ -235,9 +281,10 @@ endfunction()
 
 # Macro so we can do things in the parent scope
 macro(pf_auto)
-    _pf_auto(${ARGN})
+    _pf_auto_detect_enable_testing(${ARGN})
```

Comment:
Is this new command really necessary? Must `enable_testing()` be called before adding any tests?

Reply:
In my experimentation, yes.